MovieQuiz/Services/AlertPresenterDelegate.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  AlertPresenterDelegate.swift
+//  MovieQuiz
+//
+//  Created by beggarshrimp on 08.01.2023.
+//
+
+import UIKit
+
+protocol AlertPresenterDelegate: AnyObject {
+    func present(_ viewControllerToPresent: UIViewController, animated flag: Bool, completion: (() -> Void)?)
```

Comment:
Здесь на самом деле лучше явно передать UIAlertController, потому что узкая специализация специализация интерфейса гораздо лучше более широкой (Принципы SOLID)

```suggestion
    func present(_ alert: UIAlertController)
```
Заменив декларацию метода здесь, ее потребуется реализовать во вью контроллеере